CHANGELOG.md

```diff
@@ -10,9 +10,13 @@ This release includes version 0.102.0 of the upstream Collector components.
 
 The individual upstream Collector changelogs can be found here:
 
+0.102.1:
+
+- <https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.102.1>
+
 0.102.0:
-- https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.102.0
-- https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.102.0
+
+- <https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.102.0>
 
 <details>
 <summary>Highlights from the upstream Collector changelog</summary>
```

Comment:
This section hasn't been updated yet to reflect the 0.102.1 configgrpc entry (and the updated wording for confighttp which now mentions the upstream collector CVE).